app/tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
+    "downlevelIteration": true,
```

Comment:
Added this to manage interation on carousel slider state

Reply:
ok